eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,98 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <!--
+      To ensure that source-building .NET is successful, prebuilt package detection is enabled
+      in all contributing repos.
+
+      This repo doesn't use Arcade infra and does not support dependency flow.
+      As a result, prebuilt detection and resolution is handled differently.
+
+      Repo owners or contributors:
+      If you get an error in source-build leg pointing at new prebuilt package being detected,
+      resolve the prebuilt issue before merging your PR.
+      If that is not possible, do the following:
+        - contact source-build team
+        - add required UsagePattern entry as provided by source-build team
```

Comment:
I tend to think we should exclude anything besides resolving the prebuilt and contacting the source-build team.  I fear that suggesting creating UsagePatterns is going to lead to the problem we are trying to prevent.

Reply:
Makes sense - we want source-build team to provide guidance on a case-by-case basis. Will update - thanks.

Reply:
Resolved in https://github.com/NuGet/NuGet.Client/pull/5190/commits/eaaaebcd571a201b0bda7ae3e49767824975471a